src/schema/objects/metadata.yaml

```diff
@@ -1053,18 +1078,39 @@ GeneratedBy:
   minItems: 1
   items:
     type: object
+    required_fields: [Name]
+    recommended_fields: [Version]
```

Comment:
This PR introduces the convention of `required_fields` and `recommended_fields`. This was the rough consensus of the recent schema sprint.

Reply:
groovy.